Index: b/test/cmp.sh
===================================================================
--- a/test/cmp.sh
+++ b/test/cmp.sh
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-cmpdump.py <<EOT
+$PYINTERP -m cmpdump <<EOT
 MIITuTCCARECAQKkWTBXMQswCQYDVQQGEwJUUjEQMA4GA1UEChMHRS1HdXZlbjEUMBIGA1UECxML
 VHJ1c3RDZW50ZXIxIDAeBgNVBAMTF1JTQSBTZWN1cml0eSBDTVAgU2VydmVypC0wKzELMAkGA1UE
 BhMCVFIxHDAaBgNVBAMME1ZhbGltby1WZXR0b3ItMTdEZWOgERgPMjAxMjA1MDMxMTE2MTdaoQ8w
Index: b/test/crl.sh
===================================================================
--- a/test/crl.sh
+++ b/test/crl.sh
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-crldump.py <<EOT
+$PYINTERP -m crldump <<EOT
 -----BEGIN X509 CRL-----
 MIIBVjCBwAIBATANBgkqhkiG9w0BAQUFADB+MQswCQYDVQQGEwJBVTETMBEGA1UE
 CBMKU29tZS1TdGF0ZTEhMB8GA1UEChMYSW50ZXJuZXQgV2lkZ2l0cyBQdHkgTHRk
Index: b/test/crmf.sh
===================================================================
--- a/test/crmf.sh
+++ b/test/crmf.sh
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-crmfdump.py <<EOT
+$PYINTERP -m crmfdump <<EOT
 MIIBozCCAZ8wggEFAgUAwTnj2jCByoABAqURMA8xDTALBgNVBAMTBHVzZXKmgZ8w
 DQYJKoZIhvcNAQEBBQADgY0AMIGJAoGBAJ6ZQ2cYbn/lFsmBOlRltbRbFQUvvE0Q
 nbopOu1kC7Bmaaz7QTx8nxeiHi4m7uxCbGGxHNoGCt7EmdG8eZUBNAcHyGlXrJdm
Index: b/test/ocspreq.sh
===================================================================
--- a/test/ocspreq.sh
+++ b/test/ocspreq.sh
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-ocspreqdump.py <<EOT
+$PYINTERP -m ocspreqdump <<EOT
 MGowaDBBMD8wPTAJBgUrDgMCGgUABBS3ZrMV9C5Dko03aH13cEZeppg3wgQUkqR1LKSevoFE63n8
 isWVpesQdXMCBDXe9M+iIzAhMB8GCSsGAQUFBzABAgQSBBBjdJOiIW9EKJGELNNf/rdA
 EOT
Index: b/test/ocsprsp.sh
===================================================================
--- a/test/ocsprsp.sh
+++ b/test/ocsprsp.sh
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-ocsprspdump.py <<EOT
+$PYINTERP -m ocsprspdump <<EOT
 MIIEvQoBAKCCBLYwggSyBgkrBgEFBQcwAQEEggSjMIIEnzCCAQ+hgYAwfjELMAkGA1UEBhMCQVUx
 EzARBgNVBAgTClNvbWUtU3RhdGUxITAfBgNVBAoTGEludGVybmV0IFdpZGdpdHMgUHR5IEx0ZDEV
 MBMGA1UEAxMMc25tcGxhYnMuY29tMSAwHgYJKoZIhvcNAQkBFhFpbmZvQHNubXBsYWJzLmNvbRgP
Index: b/test/pkcs1.sh
===================================================================
--- a/test/pkcs1.sh
+++ b/test/pkcs1.sh
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-pkcs1dump.py <<EOT
+$PYINTERP -m pkcs1dump <<EOT
 -----BEGIN DSA PRIVATE KEY-----
 MIIBugIBAAKBgQCN91+Cma8UPw09gjwP9WOJCdpv3mv3/qFqzgiODGZx0Q002iTl
 1dq36m5TsWYFEcMCEyC3tFuoQ0mGq5zUUOmJvHCIPufs0g8Av0fhY77uFqneHHUi
Index: b/test/pkcs10.sh
===================================================================
--- a/test/pkcs10.sh
+++ b/test/pkcs10.sh
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-pkcs10dump.py <<EOT
+$PYINTERP -m pkcs10dump <<EOT
 -----BEGIN CERTIFICATE REQUEST-----
 MIIDATCCAekCAQAwgZkxCzAJBgNVBAYTAlJVMRYwFAYDVQQIEw1Nb3Njb3cgUmVn
 aW9uMQ8wDQYDVQQHEwZNb3Njb3cxGjAYBgNVBAoTEVNOTVAgTGFib3JhdG9yaWVz
Index: b/test/pkcs7.sh
===================================================================
--- a/test/pkcs7.sh
+++ b/test/pkcs7.sh
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-pkcs7dump.py <<EOT
+$PYINTERP -m pkcs7dump <<EOT
 -----BEGIN PKCS7-----
 MIIKdQYJKoZIhvcNAQcCoIIKZjCCCmICAQExADALBgkqhkiG9w0BBwGgggpIMIIC
 XjCCAcegAwIBAgIBADANBgkqhkiG9w0BAQQFADB1MQswCQYDVQQGEwJSVTEPMA0G
Index: b/test/pkcs8.sh
===================================================================
--- a/test/pkcs8.sh
+++ b/test/pkcs8.sh
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-pkcs8dump.py <<EOT
+$PYINTERP -m pkcs8dump <<EOT
 -----BEGIN ENCRYPTED PRIVATE KEY-----
 MIIBgTAbBgkqhkiG9w0BBQMwDgQIdtFgDWnipT8CAggABIIBYN0hkm2xqkTCt8dJ
 iZS8+HNiyHxy8g+rmWSXv/i+bTHFUReZA2GINtTRUkWpXqWcSHxNslgf7QdfgbVJ
Index: b/test/x509dump.sh
===================================================================
--- a/test/x509dump.sh
+++ b/test/x509dump.sh
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-x509dump.py <<EOT
+$PYINTERP -m x509dump <<EOT
 -----BEGIN CERTIFICATE-----
 MIIC5zCCAlACAQEwDQYJKoZIhvcNAQEFBQAwgbsxJDAiBgNVBAcTG1ZhbGlDZXJ0
 IFZhbGlkYXRpb24gTmV0d29yazEXMBUGA1UEChMOVmFsaUNlcnQsIEluYy4xNTAz
